# The ID of the shipping_box or tray the shipment was detected in
string briefcase_id

# Collection of products
DetectedConnectedProduct[] products